pkg/api/types.go

```diff
@@ -2312,6 +2324,7 @@ func (cs *ContainerService) GetProvisionScriptParametersCommon(input ProvisionSc
 		"NETWORK_API_VERSION":                  APIVersionNetwork,
 		"NETWORK_MODE":                         kubernetesConfig.NetworkMode,
 		"KUBE_BINARY_URL":                      kubernetesConfig.CustomKubeBinaryURL,
+		"MASTER_SSH_ALTERNATIVE_PORT":          cs.Properties.GetMasterSSHAlternativePort(),
```

Comment:
should this only be set on master nodes? `GetProvisionScriptParametersCommon` apply to both masters and agents